expression/expression.go

```diff
@@ -697,7 +697,9 @@ func EvaluateExprWithNull(ctx sessionctx.Context, schema *Schema, expr Expressio
 		for i, arg := range x.GetArgs() {
 			args[i] = EvaluateExprWithNull(ctx, schema, arg)
 		}
-		return NewFunctionInternal(ctx, x.FuncName.L, x.RetType, args...)
```

Comment:
Actually there's not only one place the `NewFunctionInternal` can be affected. Is it possible to check and replace them too in this pr?